common/src/main/java/io/netty/util/internal/PlatformDependent.java

```diff
@@ -155,11 +156,22 @@ public Random current() {
             }
         }
         DIRECT_MEMORY_LIMIT = maxDirectMemory;
-        logger.debug("io.netty.maxDirectMemory: {} bytes", maxDirectMemory);
+        logger.debug("-Dio.netty.maxDirectMemory: {} bytes", maxDirectMemory);
+
+        int tryAllocateUninitializedArray =
+                SystemPropertyUtil.getInt("io.netty.tryAllocateUninitializedArray", 1024);
```

Comment:
nit: If you rename as suggested above, you'll probably want to rename the property to be consistent.